store/generated_key/eigenda/verify/cert.go

```diff
@@ -34,9 +40,12 @@ type CertVerifier struct {
 	// (typically 64 blocks in happy case).
 	ethConfirmationDepth uint64
 	waitForFinalization  bool
-	manager              *edsm_binding.ContractEigenDAServiceManagerCaller
-	ethClient            *ethclient.Client
-	// The two fields below are fetched from the EigenDAServiceManager contract in the constructor.
+	svcManagerCaller     *edsm_binding.ContractEigenDAServiceManagerCaller
+
+	securityParamReader SecurityParamReader
```

Comment:
so iiuc in the previous setting the securityParams were read from the serviceManager which was inheriting (?) a thresholdRegistry.

Whereas this new Reader you have here is meant to read from the new immutable ThresholdRegistry you just merged to the monorepo? So this change is not backwards compatible with the previous architecture? Should we mention that and say that this is only compatible with your new PR code? Or am I misunderstanding this

Reply:
> So this change is not backwards compatible with the previous architecture?

assuming the backwards architecture is standard v1 verification using Service Manager then yes it is!